docs/developer/watcher-local-test.md

```diff
@@ -1,34 +1,34 @@
 # Test watcher locally in two-cluster-setup using k3d
 
 ### KCP cluster setup
-   
+
 1. Create a local control-plane (KCP) cluster:
-    ```shell
-    k3d cluster create kcp-local --port 9080:80@loadbalancer \
-    --registry-create k3d-registry.localhost:0.0.0.0:5111 \
-    --k3s-arg '--no-deploy=traefik@server:0'
-    ```
-   
+   ```shell
```

Comment:
is it possible to replace this with a kyma provision k3d command?

Reply:
Reverted back to using k3d because `kyma provision k3d` has some bugs.
I created an issue on the cli repo https://github.com/kyma-project/cli/issues/1510